docs/developer-guide/ksqldb-reference/create-stream.md

```diff
@@ -20,6 +20,15 @@ Create a new stream with the specified columns and properties.
 If the IF NOT EXISTS clause is present, the statement won't fail if a 
 stream with the same name already exists.
 
+### Source streams
+
+A stream can be created as read-only if the SOURCE clause is provided. INSERTS statements and the
+DELETE TOPIC on DROP STREAM statements won't be permitted. Source streams do not support pull
+queries currently. Only source tables support running pull queries on them.

```

Comment:
Pull queries don't work on any stream, so I'm not sure if we need to specify this here.